infrastructures/infrastructure-aws-ec2/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/AWSEC2CustomizableParameter.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.resourcemanager.nodesource.infrastructure;
+
+public class AWSEC2CustomizableParameter {
+
+    private String image;
+
+    private String vmUsername;
+
+    private String vmKeyPairName;
+
+    private String vmPrivateKey;
+
+    private int ram;
+
+    private int cores;
+
+    private String additionalProperties;
+
+    public AWSEC2CustomizableParameter(String image, String vmUsername, String vmKeyPairName, String vmPrivateKey,
```

Comment:
Yes, they should be added later. I plan to do it in the next PR. Since these two parameters are not yet supported in the current AWSEC2Infrastructure it would be a bit more complex to support them.